platform/android/doc_classes/EditorExportPlatformAndroid.xml

```diff
@@ -59,14 +59,17 @@
 		<member name="keystore/debug" type="String" setter="" getter="">
 			Path of the debug keystore file.
 			Can be overridden with the environment variable [code]GODOT_ANDROID_KEYSTORE_DEBUG_PATH[/code].
+			Fallbacks to [code]EditorSettings.export/android/debug_keystore[/code] if empty.
```

Comment:
I was going to suggest using `[member ...]` syntax, but those aren't in the XML. Do you know why?

Reply:
Here's the thing, they **are**, but they are being ignored for some reason. I bought that on the dev chat but nobody seemed to care. :stuck_out_tongue:

Reply:
I've opened an issue for it: #77878

Reply:
These ones specifically don't seem present in the XML though, or my search fu failed me. So we might have two bugs.

Reply:
I assure you they are quite real:

![image](https://github.com/godotengine/godot/assets/30739239/06e73447-2293-41f6-8323-8c4f8a13cd1e)

It's a double whammy! :tada:

Reply:
I mean `export/android/debug_keystore` doesn't exist in `doc/classes/EditorSettings.xml`, unlike other editor settings. So `[member EditorSettings.export/android/debug_keystore]` would fail, which is a bug, as the setting does exist at runtime.

Reply:
@akien-mga Yes, I agreed with you. That's why I wrote "double whammy".